giskard-frontend/src/utils.ts

```diff
@@ -50,6 +50,15 @@ export function anonymize(obj: any) {
     }
 }
 
+/** 
+ * Returning the n first chars of s ... n last chars of s
+ */
+export function sliceStr(s: string, n: number, max_size: number){
```

Comment:
@thomasabou01 , this comment is still valid. 

In fact, I think that signature of this function should be changed, the two important arguments are:
- original string (let's rename it to "original" since it's more expressive as "s")
- max_size - a desired length of a string (could also be renamed to a "max_length", but it's a nitpick)

then depending on the inputs the method will do the necessary to modify the original value to fit the desired length

Reply:
Ok yes it's logical don't know why I why complicating things

Reply:
Still few comments:
`if (max_length < 6) return s;`

1. there should rather be 5, not 6, since "..." + 2 letters = 5 chars
2. instead of returning s in case max_length is too small it's better to return an abbreviated string with 5 characters, it's not going to be as short as desired, but still better than doing nothing. So you can just replace this first line by 
```javascript
max_length = Math.max(max_length, 5);
```

Otherwise this method looks good to me